x-pack/plugins/lists/server/services/exception_lists/get_exception_list_summary.ts

```diff
@@ -84,7 +87,12 @@ export const getExceptionListSummary = async ({
     (acc, item: ByOsAggBucketType) => ({
       ...acc,
       [item.key]: item.doc_count,
-      total: acc.total + item.doc_count,
+      total:
+        // Do not add up the items by OS if host isolation exception
+        // As each host exception entry applies to all OSs
```

Comment:
thanks for commenting the "why" here :+1:

Reply:
😄